cypress/lib/util.ts

```diff
@@ -76,11 +80,54 @@ export const fakeLogin = (subscriber = true) => {
 		'https://members-data-api.theguardian.com/user-attributes/me',
 		response,
 	).as('userData');
-}
+};
 
- export const fakeLogOut = () => {
+export const fakeLogOut = () => {
 	// we can't just click sign out because cypress does not like following links to other domains
 	cy.clearCookie('GU_U');
 
 	cy.reload();
-}
+};
+
+export const mockIntersectionObserver = (win: Window, selector?: string) => {
```

Comment:
Great idea to mock the intersection observer rather than trying to trigger ads loading by scrolling to them / past them!
Might be worth giving this function a doc string explaining the parameters and behaviour when `selector` is defined or undefined.